WooCommerce/src/main/kotlin/com/woocommerce/android/ui/dashboard/google/DashboardGoogleAdsCard.kt

```diff
@@ -0,0 +1,327 @@
+    val roundedShape = RoundedCornerShape(dimensionResource(id = R.dimen.minor_100))
+    Column {
+        Row(
+            verticalAlignment = Alignment.Top,
+            modifier = modifier
+                .border(
+                    width = dimensionResource(id = R.dimen.minor_10),
+                    color = colorResource(id = R.color.divider_color),
+                    shape = roundedShape
+                )
+                .clip(roundedShape)
+                .padding(
+                    horizontal = dimensionResource(id = R.dimen.major_100),
+                    vertical = dimensionResource(id = R.dimen.major_100)
+                )
+                .background(MaterialTheme.colors.surface)
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.google_logo),
+                contentDescription = null,
+                modifier = Modifier.size(dimensionResource(id = R.dimen.major_250))
+            )
+            Column(
+                modifier = Modifier
+                    .padding(start = dimensionResource(id = R.dimen.major_100))
+                    .weight(1f),
+            ) {
+                Row {
+                    Text(
+                        text = stringResource(R.string.dashboard_google_ads_card_has_campaign_heading),
+                        style = MaterialTheme.typography.body1,
+                        fontWeight = FontWeight.SemiBold,
+                        modifier = Modifier.weight(1f)
+                    )
+
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_arrow_right),
+                        contentDescription = null
+                    )
+                }
+                Spacer(modifier = Modifier.height(dimensionResource(R.dimen.major_100)))
+
+                Row {
+                    Column(modifier = Modifier.weight(1f)) {
+                        Text(
+                            text = stringResource(R.string.dashboard_google_ads_card_has_campaign_impressions),
+                            style = MaterialTheme.typography.body2
+                        )
+                        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.minor_100)))
+                        Text(
+                            text = "1234567890",
+                            style = MaterialTheme.typography.h5,
+                            fontWeight = FontWeight.SemiBold
+                        )
+                    }
+
+                    Spacer(modifier = Modifier.width(dimensionResource(id = R.dimen.major_100)))
+
+                    Column(modifier = Modifier.weight(1f)) {
+                        Text(
+                            text = stringResource(R.string.dashboard_google_ads_card_has_campaign_clicks),
+                            style = MaterialTheme.typography.body2
+                        )
+                        Spacer(modifier = Modifier.width(dimensionResource(id = R.dimen.major_100)))
```

Comment:
Minor np, this is setting `width` instead of `height`. And is using the incorrect dimen causing missalignment between the 2 columns.  

![Screenshot 2024-07-16 at 09 24 53](https://github.com/user-attachments/assets/09277267-8145-462b-bbf3-4504161c4e95)

```suggestion
                        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.minor_100)))
```

Reply:
Thanks! Good eye.